router/batchrouter/batchrouter.go

```diff
@@ -254,6 +254,10 @@ func (brt *HandleT) postToWarehouse(batchJobs BatchJobsT, output StorageUploadOu
 		for columnName, columnType := range columns {
 			if _, ok := schemaMap[tableName][columnName]; !ok {
 				schemaMap[tableName][columnName] = columnType
+			} else {
+				if columnType == "text" {
```

Comment:
add a comment here on why we do it? and it should be set to text only if the previous value is string/text?